packages/fxa-content-server/app/scripts/templates/index.mustache

```diff
@@ -26,7 +26,7 @@
       </div>
 
       <div class="tos-pp faint">
-        {{#unsafeTranslate}}By proceeding, you agree to the <a id="fxa-tos" href="/legal/terms">Terms of Service</a> and <a id="fxa-pp" href="/legal/privacy">Privacy Notice</a>.{{/unsafeTranslate}}
+        {{#unsafeTranslate}}By proceeding, you agree to the <a id="fxa-tos" href="/legal/terms" tabindex="-1">Terms of Service</a> and <a id="fxa-pp" href="/legal/privacy" tabindex="-1">Privacy Notice</a>.{{/unsafeTranslate}}
```

Comment:
the fix will do , however i think it will trigger a retranslate of the strings? 

is there a way to dynamically add tabindexes without making translators add this?
the other way might be to run a script in the l10n repo to find replace this as well. (I hope Pontoon will be able to reflect the changes)

Reply:
wow, these strings still work like that? 🤦‍♂️

I'd prefer to do a commit to l10n that adds it over setting it in code

Reply:
It'd still be a string change on this occasion, but if we switched to the convention we use elsewhere of making the attributes data, it would mean future changes don't suffer the same problem:

```
~/c/f/p/fxa-content-server (master) $ git grep '%(escaped'
app/scripts/lib/mailcheck.js:  'Did you mean <span %(escapedEmailSuggestionAttrs)s>%(escapedDomain)s</span>?'
app/scripts/templates/confirm.mustache:        <p class="verification-email-message">{{#unsafeTranslate}}Check %(escapedEmail)s for a confirmation link and we’ll get you on your way.{{/unsafeTranslate}}</p>
app/scripts/templates/confirm.mustache:        <p class="verification-email-message">{{#unsafeTranslate}}Check your email for the verification link sent&nbsp;to %(escapedEmail)s{{/unsafeTranslate}}</p>
app/scripts/templates/confirm.mustache:    <p class="verification-email-message">{{#unsafeTranslate}}Check your email for the sign-in confirmation link sent to %(escapedEmail)s{{/unsafeTranslate}}</p>
app/scripts/templates/partial/password-strength-balloon.mustache:      >{{#unsafeTranslate}}Must not match this <a %(escapedCommonPasswordLinkAttrs)s>list of common passwords</a>{{/unsafeTranslate}}</li>
app/scripts/templates/partial/sync-suggestion.mustache:      {{#unsafeTranslate}}Looking for Firefox Sync? <a %(escapedSyncSuggestionAttrs)s>Get started here</a>{{/unsafeTranslate}}
app/scripts/templates/settings.mustache:      {{#unsafeTranslate}}Your credit card has expired. Please update it <a %(escapedCcExpiredLinkAttrs)s>here</a>.{{/unsafeTranslate}}
app/scripts/templates/settings/account_recovery/account_recovery.mustache:            <p>{{#unsafeTranslate}}<span class="note">NOTE: </span>Your Firefox Account password encrypts your Firefox data. Only your password or recovery key can access your data. <a %(escapedLearnMoreLinkAttributes)s>Learn more</a>{{/unsafeTranslate}}</p>
app/scripts/templates/settings/two_step_authentication.mustache:            {{#unsafeTranslate}}Add a layer of security to your account by requiring security codes from one of <a %(escapedTotpSupportAttributes)s>these authentication apps</a>.{{/unsafeTranslate}}
app/scripts/templates/sign_in_bounced.mustache:    <p class="faint">{{#unsafeTranslate}}If this is a valid email address, <a %(escapedSupportLinkAttrs)s>let us know</a> and we can help unlock your account.{{/unsafeTranslate}}</p>
app/scripts/templates/sms_send.mustache:      {{#unsafeTranslate}}SMS service only available in certain countries. SMS &amp; data rates may apply. The intended recipient of the email or SMS must have consented. <a %(escapedLearnMoreAttributes)s>Learn more</a>{{/unsafeTranslate}}
app/scripts/templates/sms_sent.mustache:        {{#t}}A link to the Firefox app was sent to %(escapedPhoneNumber)s{{/t}}
app/scripts/templates/sms_sent.mustache:            {{#unsafeTranslate}}App link sent to %(escapedPhoneNumber)s. <a %(escapedBackLinkAttrs)s>Mistyped&nbsp;number?</a>{{/unsafeTranslate}}
app/scripts/templates/sms_sent.mustache:            {{#unsafeTranslate}}App link resent to %(escapedPhoneNumber)s. <a %(escapedBackLinkAttrs)s>Mistyped&nbsp;number?</a>{{/unsafeTranslate}}
app/tests/spec/lib/strings.js:        Strings.hasUnsafeVariables('this has a %(escapedVariable)s')
app/tests/spec/lib/strings.js:          'this has multiple %(escapedVariable)s, here %(escapedToo)s'
app/tests/spec/lib/strings.js:          'this has both %(escapedSafeVariable)s and %(unsafeVariable)s'
```

Reply:
🤢

Reply:
i think i'd rather do a find & replace on the l10n repo tbh